internal/logger/logger.go

```diff
@@ -0,0 +1,29 @@
+package logger
+
+import (
+	"io"
+	"log/slog"
+)
+
+func Init(config *Config, output io.Writer) {
+	var programLevel = new(slog.LevelVar)
+
+	switch config.Verbosity {
+	case 0:
+		programLevel.Set(slog.LevelInfo)
+	case 1:
+		programLevel.Set(slog.LevelWarn)
+	case 2:
+		programLevel.Set(slog.LevelDebug)
+	case 3:
+		programLevel.Set(slog.LevelError)
+	default:
+		programLevel.Set(slog.LevelInfo)
```

Comment:
I think default log level should be error. I don't want logging to be too verbose unless asked. The more `-v` flags, the more verbose — i.e. from error to debug. Let's start out at the least level of verbosity by default.